tests/rptest/services/redpanda.py

```diff
@@ -992,6 +992,60 @@ def start_rp():
         self.start_service(node, start_rp)
         self._started.append(node)
 
+    def start_node_with_rpk(self, node, additional_args=""):
```

Comment:
I see this method is very similar to `start_node`. I've vote for breaking out similar components into a private method, to reduce code duplication

Reply:
Got it! moved rpk method to its own client and made is_status_ready a private method.